media/src/test/java/io/wcm/handler/media/markup/SimpleImageMediaMarkupBuilderTest.java

```diff
@@ -384,7 +406,7 @@ private void assertMap(HtmlElement<?> element) {
   void testIsValidMedia_Image() {
     MediaMarkupBuilder builder = AdaptTo.notNull(context.request(), SimpleImageMediaMarkupBuilder.class);
 
-    assertFalse(builder.isValidMedia(null));
+    assertFalse(builder.isValidMedia(null)); // FIXME: the parameter 'element' has @NotNull annotation and cannot be null
```

Comment:
can we resolve this FIXME? if i remember correctly this check is only for backwards compatibility as in early releases the method did not have a @NonNull annotation
same applies to next FIXME in next test case

Reply:
replaced FIXME with a comment